README.md

```diff
@@ -91,32 +92,31 @@ You can also filter the containers for which logs/stats are forwarded.
 ##### Notes
 
 - Please ensure you correctly escape the regex pattern since these can be expanded/interpreted by your shell; ideally in single-quotes.
-  * Use `'.*nginx.*'` rather than `.*nginx.*`
+  - Use `'.*nginx.*'` rather than `.*nginx.*`
 - Do not supply the REGEX pattern in a LEQL fashion as in the UI, filtering is based on normal regular expressions.
-  * Use `'.*nginx.*'`, not `'/.*nginx.*/'`, note the removed foreslashes.
+  - Use `'.*nginx.*'`, not `'/.*nginx.*/'`, note the removed foreslashes.
 - For each of the arguments below you can only supply at most one of each.
-  * Use `--skipByName '.*(nginx|haproxy).*'`
-  * Do not use
+  - Use `--skipByName '.*(nginx|haproxy).*'`
+  - Do not use
+
     ```bash
     --skipByName '.*nginx.*' --skipByName '.*haproxy.*'
     ```
 
 ##### Filters
 
-* `--matchByName '<REGEX>'`: forward logs/stats only for the containers whose name matches the given `<REGEX>`.
-* `--matchByImage '<REGEX>'`: forward logs/stats only for the containers whose image matches the given `<REGEX>`.
-* `--skipByName '<REGEX>'`: do not forward logs/stats for the containers whose name matches the given `<REGEX>`.
-* `--skipByImage '<REGEX>'`: do not forward logs/stats for the containers whose image matches the given `<REGEX>`.
-
+- `--matchByName '<REGEX>'`: forward logs/stats only for the containers whose name matches the given `<REGEX>`.
+- `--matchByImage '<REGEX>'`: forward logs/stats only for the containers whose image matches the given `<REGEX>`.
+- `--skipByName '<REGEX>'`: do not forward logs/stats for the containers whose name matches the given `<REGEX>`.
+- `--skipByImage '<REGEX>'`: do not forward logs/stats for the containers whose image matches the given `<REGEX>`.
 
 ## Usage as a Container
 
 The simplest way to forward all your container logs to the Rapid7 Insight Platform is to run this repository as a container.
 
 - If you are using this container in production please ensure that you pin the version of the image.
-  * Use `rapid7/r7insight_docker:3.1.3` rather than `rapid7/r7insight_docker:latest` since `latest` might pull in breaking changes if a new version is released
-  * You can see the the available versions [here](https://hub.docker.com/r/rapid7/r7insight_docker/tags).
-
+  - Use `rapid7/r7insight_docker:3.1.3` rather than `rapid7/r7insight_docker:latest` since `latest` might pull in breaking changes if a new version is released
```

Comment:
prob bump the version number in this sentence too since you're touching it anyway :D

Reply:
can't be arsed maintaining the version number so went for a placeholder